api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -709,23 +709,23 @@ func (m *AzureManagedControlPlane) validateVirtualNetworkUpdate(old *AzureManage
 	if old.Spec.VirtualNetwork.Name != m.Spec.VirtualNetwork.Name {
 		allErrs = append(allErrs,
 			field.Invalid(
-				field.NewPath("Spec", "VirtualNetwork.Name"),
+				field.NewPath("spec", "virtualNetwork.Name"),
 				m.Spec.VirtualNetwork.Name,
 				"Virtual Network Name is immutable"))
 	}
 
 	if old.Spec.VirtualNetwork.CIDRBlock != m.Spec.VirtualNetwork.CIDRBlock {
 		allErrs = append(allErrs,
 			field.Invalid(
-				field.NewPath("Spec", "VirtualNetwork.CIDRBlock"),
+				field.NewPath("spec", "virtualNetwork.CIDRBlock"),
 				m.Spec.VirtualNetwork.CIDRBlock,
 				"Virtual Network CIDRBlock is immutable"))
 	}
 
 	if old.Spec.VirtualNetwork.Subnet.Name != m.Spec.VirtualNetwork.Subnet.Name {
 		allErrs = append(allErrs,
 			field.Invalid(
-				field.NewPath("Spec", "VirtualNetwork.Subnet.Name"),
+				field.NewPath("spec", "virtualNetwork.Subnet.Name"),
```

Comment:
```suggestion
				field.NewPath("spec", "virtualNetwork.subnet.name"),
```
Same for the rest of the paths in this file.